Dockerfile

```diff
@@ -14,6 +14,9 @@ USER airflow
 COPY requirements.build.txt ./
 RUN pip install --disable-pip-version-check -r requirements.build.txt
 
+COPY requirements.monitoring.txt ./
+RUN pip install --disable-pip-version-check -r requirements.monitoring.txt
```

Comment:
Maybe just add any requirements you may need for monitoring to `requirements.txt` or do you see an advantage in keeping it separate?

The `requirements.build.txt` for example are separate because the version of those dependencies (including `pip`) can affect the installation of the other dependencies.

Reply:
Because the requirements list is very long already I wanted to keep it separate in case of we want to move monitoring in another repo in the future as it is not a data pipeline and the job for checking every pipeline we have.